src/emit/ops.rs

```diff
@@ -400,7 +400,7 @@ pub(crate) fn emit_custom_unary_op<'c, H, F>(
 where
     H: HugrView,
     F: FnOnce(
-        &Builder<'c>,
+        &mut EmitFuncContext<'c, H>,
```

Comment:
This was necessary to call `emit_value` from the continuation - should I update `emit_custom_binary_op` to be consistent?

Reply:
Yes, better to be consistent 👍